lib/components/artist/related_artists/related_artist.js

```diff
@@ -35,7 +35,7 @@ class RelatedArtist extends React.Component {
     }
 
     const totalWorks = counts.artworks ? counts.artworks + (counts.artworks > 1 ? ' works' : ' work') : null
-    if (counts.for_sale_artworks === counts.artworks) {
+    if (totalWorks && counts.for_sale_artworks === counts.artworks) {
```

Comment:
const totalWorks is a `?string` here because of the nulls, what do you get when you add `null` to a string?

```js
> null + 'thing'
"nullthing"
```

hah. So we ensure there's a total works object there before using it